### PR TITLE
Add logs to try to diagnose #17181

### PR DESCRIPTION
**Pull request by thockin**
Forward progress on #17181

### CODE REVIEW

File: pkg/kubelet/network/exec/exec_test.go [hunk 1]
```diff
@@ -313,19 +313,22 @@ func TestPluginStatusHookIPv6(t *testing.T) {
...
 	ip, err := plug.Status("namespace", "name", "dockerid2345")
 	if err != nil {
-		t.Errorf("Expected nil got %v", err)
+		t.Errorf("Status() failed: %v", err)
 	}
 	// check output of status hook
 	output, err := ioutil.ReadFile(path.Join(testPluginPath, pluginName, pluginName+".out"))
 	if err != nil {
-		t.Errorf("Expected nil")
+		t.Errorf("ReadFile failed: %v", err)
```
Comment (brendandburns) [inline pkg/kubelet/network/exec/exec_test.go]: print path here too?

**Comment by brendandburns** [comment]
LGTM, one further thought on an added log info.  optional.  feel free to self-LGTM again

File: pkg/kubelet/network/exec/exec_test.go (hunk 1 above)
Comment (thockin) [inline pkg/kubelet/network/exec/exec_test.go]: done and pushed

**Comment by thockin** [comment]
@k8s-oncall consider for manual merge for test flakes (it's just more logs, so use judgement)

**Comment by dchen1107** [comment]
ACK in my queue now. Just merged several ones. Will do this once build is done for previous ones.

**Comment by dchen1107** [comment]
Jenkins kubernetes-pull-build-test-e2e-gce cannot make much progress due to many re-run and pending prs. This one is pretty simple and useful for our e2e debugging. I am merging it without gce e2e finish here.

**Pull request merged**